tool/debian/build.sh

```diff
@@ -86,26 +86,22 @@ if [ -n "$BUILD_ID" ]; then
 fi
 
 FOLDER=$FOLDER_PREFIX-cpp
-if [ $CUDA = "ON" ]; then
-  FOLDER=$FOLDER_PREFIX-cuda$CUDA_VERSION-cudnn$CUDNN_VERSION
-fi
-
 if [ $PYTHON = "ON" ]
 then
   FOLDER=$FOLDER/python-singa
```

Comment:
need to update the path?
the path for python part has been changed from python-singa to singa-cpp/python-singa/